src/signalSTM.h

```diff
@@ -0,0 +1,365 @@
+/*
+*   RF_RECEIVER v3.4.0 for Arduino
+*   Sketch to use an arduino as a receiver/sending device for digital signals
+*
+*   The Sketch can also encode and send data via a transmitter,
+*   while only PT2262 type-signals for Intertechno devices are implemented in the sketch,
+*   there is an option to send almost any data over a send raw interface
+*   2014-2015  N.Butzek, S.Butzek
+*   2016-2018 S.Butzek
+
+*   This software focuses on remote sensors like weather sensors (temperature,
+*   humidity Logilink, TCM, Oregon Scientific, ...), remote controlled power switches
+*   (Intertechno, TCM, ARCtech, ...) which use encoder chips like PT2262 and
+*   EV1527-type and manchester encoder to send information in the 433MHz Band.
+*   But the sketch will also work for infrared or other medias. Even other frequencys
+*   can be used
+*
+*   This program is free software: you can redistribute it and/or modify
+*   it under the terms of the GNU General Public License as published by
+*   the Free Software Foundation, either version 3 of the License, or
+*   (at your option) any later version.
+*
+*   This program is distributed in the hope that it will be useful,
+*   but WITHOUT ANY WARRANTY; without even the implied warranty of
+*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*   GNU General Public License for more details.
+*
+*   You should have received a copy of the GNU General Public License
+*   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+
+// See config flags in compile_config.h:
+#include "compile_config.h"
+
+#if !defined(__AVR__) && defined(MAPLE_Mini)
+
+#define PROGNAME               " SIGNALduino_STM "
+#define PROGVERS               "3.4.0-dev_20200626"
+#define VERSION_1               0x33
+#define VERSION_2               0x1d
+
+#ifdef MAPLE_Mini
```

Comment:
Brauchen wir nicht mehr, wenn diese Datei ab der 2. Zeile auf defined(MAPLE_Mini) prüft denke ich

Reply:
Ich lies die „if Maple...“ drin weil dies erleichtert es, wenn ggf von der STM32 ein anderer Prozessor mal hinzu kommt.

Reply:
Ok bei Pins macht das dann ggf. Sinn, bei Befehlen eher weniger oder?

Reply:
Es könnte bei Befehlen sein , das ein anderer Prozessor einen anderen Befehl oder eine andere Schreibweise besitzt. (Sowas gibt es ja)

Reply:
Würde ich erst dann einbauen, wenn wir das auch wissen dass es so ist. Auf Verdacht so was einbauen hmm.

Reply:
Okay, ich gebe mich geschlagen und nehme die if Maple in der Datei heraus 😉